script/configure-web-proxy

```diff
@@ -7,20 +7,30 @@ Configure a local web proxy using apache for openQA.
 
 Options:
  -h, --help         display this help
+ -n, --nginx        setup nginx instead of apache
```

Comment:
I suggest `--proxy=nginx` defaulting to apache so that we can easily extend for more or change the default